component/src/test/java/org/wso2/extension/siddhi/io/jms/sink/JMSSinkTestCase.java

```diff
@@ -161,24 +150,83 @@ public void jmsTopicPublishTest4() throws InterruptedException {
 
             // deploying the execution plan
             SiddhiManager siddhiManager = new SiddhiManager();
-            String inStreamDefinition = "" +
-                    "@sink(type='jms', @map(type='xml'),"
+            String inStreamDefinition = "" + "@sink(type='jms', @map(type='xml'),"
                     + "factory.initial='org.wso2.activemq.jndi.ActiveMQInitialContextFactory11', "
-                    + "provider.url='vm://localhost',"
-                    + "destination='DAS_JMS_OUTPUT_TEST' "
-                    + ")" +
-                    "define stream inputStream (name string, age int, country string);";
+                    + "provider.url='vm://localhost'," + "destination='DAS_JMS_OUTPUT_TEST' " + ")"
+                    + "define stream inputStream (name string, age int, country string);";
             executionPlanRuntime = siddhiManager.
                     createSiddhiAppRuntime(inStreamDefinition);
             InputHandler inputStream = executionPlanRuntime.getInputHandler("inputStream");
             executionPlanRuntime.start();
-            inputStream.send(new Object[]{"JAMES", 23, "USA"});
-            inputStream.send(new Object[]{"MIKE", 23, "Germany"});
+            inputStream.send(new Object[] { "JAMES", "23", "USA" });
```

Comment:
I think we should send as  23 instead of "23" because its an integer. Why this is changed. and this is used in all places ?